src/fn/upload-extension-libs.js

```diff
@@ -0,0 +1,55 @@
+const util = require('util');
+const fs = require('fs');
+
+const readdir = util.promisify(fs.readdir);
+const stat = util.promisify(fs.stat);
+
+const environment = require('../lib/environment');
+const pouch = require('../lib/db');
+const log = require('../lib/log');
+const attachmentsFromDir = require('../lib/attachments-from-dir');
+const warnUploadOverwrite = require('../lib/warn-upload-overwrite');
+const insertOrReplace = require('../lib/insert-or-replace');
+
+const DIR_NAME = 'extension-libs';
```

Comment:
Should this be included in `../lib/project-paths`?

Reply:
True, done!